DependencyInjection/Compiler/CacheCompatibilityPass.php

```diff
@@ -59,7 +59,7 @@ private function updateSecondLevelCache(ContainerBuilder $container, Definition
             $factoryDefinition = $methodCall[1][0];
             assert($factoryDefinition instanceof Definition);
             $aliasId = (string) $factoryDefinition->getArgument(1);
-            $this->wrapIfNecessary($container, $aliasId, (string) $container->getAlias($aliasId), false);
+            $this->wrapIfNecessary($container, $aliasId, (string) $container->getAlias($aliasId), true);
```

Comment:
second level also supports PSR6 caches since ORM 2.11 (which we require). See https://github.com/doctrine/orm/pull/9322